### PR TITLE
koding/workers: remove non existent records

### PR DESCRIPTION
**Pull request by cihangir**

### CODE REVIEW

**Comment by cihangir** [comment]
@rjeczalik @gokmen could you guys check the overall logic & possible bugs? I dont want to create data corruption

File: go/src/koding/workers/removenonexistents/main.go [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+	"github.com/koding/cache"
+)
+
+var (
+	conf        *config.Config
+	flagProfile = flag.String("c", "prod", "Configuration profile from file")
+	flagSkip    = flag.Int("s", 0, "Configuration profile from file")
+	flagLimit   = flag.Int("l", 1000, "Configuration profile from file")
+	flagDry     = flag.Bool("dry", false, "dry run")
+	flagColls   = flag.String("colls", "jUsers,jAccounts,jWorkspaces,jNames,jComputeStacks,jCombinedAppStorages,relationships", "collections to clean up")
```
Comment (rjeczalik) [inline go/src/koding/workers/removenonexistents/main.go]: Do we use jWorkspaces at all? Can we archive them and remove all from db?

Comment (gokmen) [inline go/src/koding/workers/removenonexistents/main.go]: IDE part can be removed but we need to have a similar collection to keep collaboration data, currently it's the only requirement for this collection from IDE perspective.

**Review by gokmen — APPROVED** [review]

**Review by rjeczalik — APPROVED** [review]

**Comment by cihangir** [comment]
@szkl ping

**Pull request merged**